cron.go

```diff
@@ -37,9 +37,10 @@ type Job interface {
 
 // Schedule describes a job's duty cycle.
 type Schedule interface {
-	// Next returns the next activation time, later than the given time.
+	// Next returns the next activation time, later than the current time.
+	// If `after` time is provided then the activation time returned will be later than `after`.
 	// Next is invoked initially, and then each time the job is run.
-	Next(time.Time) time.Time
+	Next(from, after time.Time) time.Time
```

Comment:
Is that interface only used internally in that package? 
If not, consider leaving the previous method in the interface (so you will have two methods) and just call the new function from it with zero value for `after`.

Reply:
Currently the V4 version is used only in the Timetable package used by RETL. 
However felt its a good idea to maintain previous definition, so added the older Next() method as you suggested which calls the newer method.